src/Illuminate/Foundation/Console/OptimizeCommand.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace Illuminate\Foundation\Console;
+
+use Illuminate\Console\Command;
+
+class OptimizeCommand extends Command
+{
+    /**
+     * The console command name.
+     *
+     * @var string
+     */
+    protected $name = 'optimize';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Optimize everything (cache routes, config)';
+
+    /**
+     * Execute the console command.
+     *
+     * @return void
+     */
+    public function handle()
+    {
+        $this->call('cache:clear');
+        $this->call('config:cache');
+        $this->call('route:clear');
```

Comment:
`route:cache` implicitly clears the route cache before creating it again.

Reply:
Yeah, this has been removed in later commits.